src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -56,20 +57,26 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         private const string LiteralPathParameterSet = "LiteralParamSet";
         private const string InputObjParamSet = "InputObjParamSet";
         private MarkdownConversionType conversionType = MarkdownConversionType.HTML;
-        private MarkdownOptionInfo mdOption = null;
+        private PSMarkdownOptionInfo mdOption = null;
 
         /// <summary>
-        /// Read the MarkdownOptionInfo set in SessionState.
+        /// Read the PSMarkdownOptionInfo set in SessionState.
         /// </summary>
         protected override void BeginProcessing()
         {
-            mdOption = SessionState.PSVariable.GetValue("MarkdownOptionInfo", new MarkdownOptionInfo()) as MarkdownOptionInfo;
+            mdOption = SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
 
             if (mdOption == null)
             {
                 throw new InvalidOperationException();
             }
 
+            if (this.Host != null && this.Host.UI.SupportsVirtualTerminal)
```

Comment:
`this.Host.UI.SupportsVirtualTerminal` is not needed twice here.  But it seems like the logic should be:
```
mdOption.EnableVT100Encoding = mdOption.EnableVT100Encoding && (this.Host != null) && (this.Host.UI.SupportsVirtualTerminal);

If there is no host I assume EnableVT100Encoding option should be disabled?

Reply:
@PaulHigin improved it a bit by using nullable bool.